File: CRUD - PHP e JAVA em POO/PHP/model/produto.php
<?php
require_once 'conexao.php';
require_once 'iProduto.php';

class Produto implements iProduto  //Vai implementar a interface iProduto
{
    private $nome;
    private $descricao;
    private $marca;

    public function cadastrar($nome,$descricao,$marca){
       $this->nome=$nome;
       $this->descricao=$descricao;
       $this->marca=$marca;
			
			$conexao = new Conexao(); //Instancia a class Conexão
			$conexao->conectar(); // Chama o método conectar()
			
			$sql ="INSERT INTO produto (nome, descricao, marca) values ('$nome','$descricao','$marca')";
		
			mysqli_query($conexao->getCnx(),$sql);
    }
	
	 public function buscar(){
		 
		$conexao = new Conexao(); //Instancia a class Conexão
        $conexao->conectar(); // Chama o método conectar()
		$sql="SELECT * FROM produto";
		$resultado = mysqli_query($conexao->getCnx(),$sql);
		

		while ($dado = mysqli_fetch_object($resultado)){
			echo $dado->nome
			. "-" 
			. $dado->descricao 
			. "-" 
			. $dado->marca;
			echo "<a href='controller/deletrar_produto.php?id=" .$dado->id. "'> [x] </a>";
			echo "<br>";
		} 
	 }
	 
	public function deletar($id){
		$conexao = new Conexao(); //Instancia a class Conexão
        $conexao->conectar(); // Chama o método conectar()
		
		$sql="DELETE FROM produto WHERE id=$id";
		mysqli_query($conexao->getCnx(),$sql);
	}
} //End Class

File: CRUD - PHP e JAVA em POO/JAVA/produto.java
package br.com.meuprojeto;


import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;

public class produto {
	
	
	PreparedStatement instrucao = null;
    ResultSet rs = null;

    
    
    
	public void cadastrar(String nome, String desc, String marca) {
	        try {

	            instrucao = Conexao.getConexao().prepareStatement("INSERT INTO produto (nome, descricao, marca) values (?,?,?)");

	            instrucao.setString(1, nome);
	            instrucao.setString(2, desc);
	            instrucao.setString(3, marca);
	            instrucao.executeUpdate();
	            instrucao.close();
	            
	        } catch (SQLException e) {
	           e.printStackTrace();
	        }
	}
	
	
	
	public void buscar() {
		try {

		PreparedStatement resultado = Conexao.getConexao().prepareStatement("SELECT * FROM produto");
		rs = resultado.executeQuery();
		
		while(rs.next()){

		    String nome = rs.getString("nome");
		    String descricao = rs.getString("descricao");
		    String marca = rs.getString("marca");

		    System.out.print(" |" + nome + marca + descricao);
		}
		
		} catch (SQLException e) {
	           e.printStackTrace();
	        }
	}
}
